src/system.h

```diff
@@ -1031,7 +1031,11 @@ extern void SyExit (
 **  Returns -1 to represent failure
 **
 */
-Int8 SyNanosecondsSinceEpoch();
+extern Int8 SyNanosecondsSinceEpoch();
+extern Int8 SyNanosecondsSinceEpochResolution();
+
+extern const char *SyNanosecondsSinceEpochMethod;
+extern Int SyNanosecondsSinceEpochMonotonic;
```

Comment:
Perhaps make SyNanosecondsSinceEpochMonotonic `const`? And change `SyNanosecondsSinceEpochMethod` similarly to `const char * const`?